src/Take.Blip.Builder/Hosting/ContainerExtensions.cs

```diff
@@ -56,6 +56,7 @@ private static Container RegisterBuilderRoot(this Container container)
             container.RegisterSingleton<IContextProvider, ContextProvider>();
             container.RegisterDecorator<ISender, OwnerSenderDecorator>(Lifestyle.Singleton);
             container.RegisterSingleton<IUserOwnerResolver, UserOwnerResolver>();
+            container.RegisterSingleton<IInputExpirationHandler, InputExpirationHandler>();
```

Comment:
This shouldn't be a singleton since the implementation has a state (_stateIdInputExipiration) that could not be shared with other threads

Reply:
@andreminelli i remove this interval variable how @ceifa suggest.

Reply:
Sorry, I have read an old commit...